Makefile

```diff
@@ -16,6 +16,12 @@ test-functional:
 test-schemas:
 	pipenv run ./scripts/test_schemas.sh
 
+translation-templates:
+	pipenv run python -m scripts.run_translation_templates
+
+test-translation-templates:
+	pipenv run python -m scripts.run_translation_templates --check
```

Comment:
Given the command line flag is `check` this should be `check-translation-templates`.

Reply:
I actually think test is more appropriate here given the other script commands use it.